src/pages/LandingPage.js

```diff
@@ -75,11 +89,10 @@ const LandingPage = () => {
     ]),
     isTerra() && div({
       style: {
+        ...styles.callToActionBanner,
         backgroundColor: '#191a1c', // This fallback color was extracted from the left edge of the background image
-        backgroundImage: `url(${covidHero})`, backgroundSize: 'cover', borderRadius: 5,
-        boxShadow: '0 2px 5px 0 rgba(0,0,0,0.35), 0 3px 2px 0 rgba(0,0,0,0.12)',
-        color: 'white', padding: '2rem 1rem',
-        width: 'calc(675px + 2rem)' // 3 card widths + 2 card margins to line up with the content directly above
+        backgroundImage: `url(${covidHero})`,
+        width: `calc(${styles.card.width * 3}px + ${styles.card.marginRight} * 2)`
```

Comment:
I think you need to add `rem` to the end of the margin in the calc string?

Reply:
`styles.card.marginRight` is a string with `rem` units. Thus, the other operand for multiplication must be a number (without units) vs a length.

https://developer.mozilla.org/en-US/docs/Web/CSS/calc#syntax

Reply:
If you run this locally tho, it works - I wouldn't be pushing up code without testing it!